variants/m5stack_core/platformio.ini

```diff
@@ -7,7 +7,7 @@ build_src_filter =
 build_flags = 
   ${esp32_base.build_flags} -I variants/m5stack_core
   -DILI9341_DRIVER
-  -DM5STACK
+  -DM5STACK_CORE
   -DUSER_SETUP_LOADED
```

Comment:
you need to update src/platform/esp32/architecture.h and a few other places too.

Reply:
It should be OK now.